sql/versioned/V2__add_email_column.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE customers
+    ADD COLUMN email varchar(1000)
```

Comment:
Uhh is 1000 characters really necessary?

Reply:
Yes because reasons

Reply:
Oh ok that makes sense